src/app/components/common/settings-buttons.ts

```diff
@@ -68,7 +68,6 @@ export const SettingsButtonsGroups: string[][] = [
     'showMoreInfo',
     'compactView',
     'fontSizeLarger',
-    // 'randomizeGallery', // TODO - disabled for now
```

Comment:
😢

Reply:
The `shuffle current results` button is still there -- the _randomizeGallery_ button, if I remember right, was a way to ask VHA to always show a random thumbnail from the extracted ones, rather than the first screenshot.

Given that we're storing the extracted screenshots separately from the first, it would require the app to load the larger .jpg strip rather than the single preview -- and it would require I write some more code that I don't expect many would use 🤷‍♂